internal/db/dbtest/dbtest.go

```diff
@@ -68,7 +68,11 @@ func NewDB(t testing.TB, dsn string) (*sql.DB, func()) {
 	config.Path = "/" + dbname
 	testDB := dbConn(t, config)
 
-	if err = dbutil.MigrateDB(testDB, dsn); err != nil {
+	m, err := dbutil.NewMigrate(testDB, dsn)
+	if err != nil {
+		t.Fatalf("failed to construct migration: %s", err)
```

Comment:
I saw every other places use plural:
```suggestion
		t.Fatalf("failed to construct migrations: %s", err)
```